cmd/commands/set.go

```diff
@@ -0,0 +1,61 @@
+package command
+
+import (
+	"fmt"
+	"regexp"
+
+	"github.com/red-hat-storage/odf-cli/pkg/set"
+	"github.com/rook/kubectl-rook-ceph/pkg/logging"
+	"github.com/spf13/cobra"
+)
+
+// SetCmd represents the set command
+var SetCmd = &cobra.Command{
+	Use:                "set",
+	Short:              "Call subcommands like 'recover-profile'",
+	DisableFlagParsing: true,
+	Args:               cobra.ExactArgs(1),
+}
+
+var validArgs = []string{"high_client_ops", "high_recovery_ops", "balanced"}
+
+var recoverProfile = &cobra.Command{
+	Use:                "recover-profile",
+	Short:              "Update the mclock scheduler profile value `high_client_ops, high_recovery_ops, balanced` used during recovery for single osd or all the osds",
+	DisableFlagParsing: true,
+	Args:               cobra.RangeArgs(1, 2),
+	Run: func(cmd *cobra.Command, args []string) {
+		if isValidArg(args) {
+			clientsets := GetClientsets(cmd.Context())
+			set.SetProfile(cmd.Context(), clientsets, args, OperatorNamespace, StorageClusterNamespace)
+		} else {
+			logging.Error(fmt.Errorf("Invalid arguments for recover-profile command"))
+		}
+	},
+}
+
+func init() {
+	SetCmd.AddCommand(recoverProfile)
+}
+
+func isValidArg(args []string) bool {
+	for c, valid := range validArgs {
```

Comment:
Assigning a var to args[0] will really help with the readability of this method
```suggestion
	recoveryInput := args[0]
	for c, valid := range validArgs {
```